src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,14 @@
+import Article from "./Article"
+
+export default function ArticlesList({articles}){
+    return(
+        <ol>
+        { articles.length!==0?(
+            articles.articles.map((article)=>{
+              return  <Article key={article.article_id} article={article}/>
+            })
+        ):null}
```

Comment:
great logic here, i wonder if you can return a more helpful thing than null if there are no articles so the user knows the website is still working but there just isn't anything there